src/main/java/core/basesyntax/FigureDrawable.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface FigureDrawable {
+    void getInfo();
+}
```

Comment:
The method name `getInfo` in the interface `FigureDrawable` is not very descriptive of the action it should perform. Consider renaming it to something more specific, like `draw`, since this interface is meant to define drawable figures.